### DISCUSSION TITLE
How can I compile pytorch3d inference to run it in production?

### DISCUSSION THREAD

**Discussion by Alexankharin** [facebookresearch/pytorch3d#936] [Q&A]
I've trained a pytirch3d model that I want to place inference on server with GPUs (AWS). It is highly desirable for me to keep as few dependencies  as possible. Is it possible to convert the pytorch3d model (I've used only pointcloud rasterizer function from the library) into jit-traced/scripted model or to onnx format?

**Comment by SalehChoueib** [comment]
Hi Alexander, 

I was wondering if you had any luck with inferencing a Pytorch3D model with onnx?

**Reply by saidElhadi** [reply]
Hi Salah

Were you able to inference a model built with pytorch2d using onnx?
